tests/functional/job-submission/09-activity-log-host-bad-submit/suite.rc

```diff
@@ -13,20 +13,17 @@
     [[root]]
     [[bad-submitter]]
         script = true
-        [[[job]]]
-            batch system = at
-            batch submit command template = at non
-        [[[remote]]]
-            host = {{CYLC_TEST_HOST}}
+        platform = {{ CYLC_REMOTE_PLATFORM }}
     [[grepper]]
         script = """
-set -x
-# Test that the original command is printed
-A_LOG="$(dirname "$0")/../../bad-submitter/01/job-activity.log"
-grep '\[jobs-submit cmd\] cylc jobs-submit.* --host={{CYLC_TEST_HOST}}' \
-    "${A_LOG}"
-# Test that some STDERR is printed, with the host name as prefix
-grep '({{CYLC_TEST_HOST}}) .* \[STDERR\] ' "${A_LOG}"
-# Stop the suite cleanly
-cylc stop "${CYLC_SUITE_NAME}"
-"""
+            set -x
+            # Test that the original command is printed
+            A_LOG="$(dirname "$0")/../../bad-submitter/01/job-activity.log"
+            grep '\[jobs-submit cmd\] ssh .* {{CYLC_REMOTE_PLATFORM}} .* cylc jobs-submit.*' \
+                "${A_LOG}"
+            # Test that some STDERR is printed, with the host name as prefix
+            # TODO: re-instate this functionality
```

Comment:
Ah I see this is a todo to reinstate this. I'm guessing the above test 08-activity-log-host is to be reinstated at a later date then?

Reply:
By experiment, this branch does not write the `(hostname)` prefix to STDERR lines in the job activity log (unlike master).

So I think you can just remove this TODO and the comment-out grep (which if we were to reinstate the functionality should grep for the host not the platform).